WordPress/src/main/java/org/wordpress/android/ui/sitecreation/segments/NewSiteCreationSegmentViewHolder.kt

```diff
@@ -35,8 +35,10 @@ sealed class NewSiteCreationSegmentViewHolder(internal val parent: ViewGroup, @L
             title.text = uiState.title
             subtitle.text = uiState.subtitle
             imageManager.load(icon, IMAGE, uiState.iconUrl)
```

Comment:
Should we add a case to use a default icon when `iconUrl` is empty similar to `iconColor`?  The fix for `iconColor` below works, but the icon is still not visible as shown below since the `iconUrl` is empty also.

![site_creation_segments_icon](https://user-images.githubusercontent.com/3827611/50411509-7dfe8380-07ce-11e9-855c-c40f942a8db9.png)

I see one of the items in https://github.com/wordpress-mobile/WordPress-Android/issues/8864  is:
> What should we do if the `iconUrl` or `iconColor` is missing for a segment? This is currently the case for the `Online Store` segment. We have a `TODO` for this in `NewSiteCreationSegmentViewHolder` as we are just setting the color to black if it's missing, but don't do anything about the missing `iconUrl`.

Just let me know if we're deferring the default `iconUrl` value until getting input from designers.

Reply:
We are just leaving it as is since we have an issue for it. It's really not worth the effort right now, it's a very simple fix once we know what we want to do with it. Thanks for checking in again, sorry if it wasn't very clear.

P.S: Keep in mind that this is merging to a WIP branch and it's also behind a feature flag, so as long as we have an issue we are completely fine merging non-polished PRs.